CinemaTicketSalesSystem/CinemaTicketSalesBusinessLogic/Interfaces/IMovieService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using ApplicationDbMovies.Models;
+using CinemaTicketSalesBusinessLogic.Models;
+
+namespace CinemaTicketSalesBusinessLogic.Interfaces
+{
+    public interface IMovieService
+    {
+        IEnumerable<MovieInfoModel> GetMovies();
+        Movie GetMovieDetails(int id);
```

Comment:
You should not return db model here.
It should be MovieModel or MovieDetailsModel